lang/en.yml

```diff
@@ -129,6 +129,10 @@ BW:
     speedDiePenalty: Speed Die Penalty
     climbingPenalty: Climbing Penalty
     swimmingPenalty: Swimming Penalty
+  possession:
+    rpCost: Resources Point Cost
```

Comment:
This one is going to appear in a bunch of different item sheets. Probably worth refactoring to a common key soon.